### PR TITLE
Created folder and page for Eth2 Staking Community Grants

### PR DESCRIPTION
**Pull request by gilbertginsberg**
## Description

Set up base for Eth2 Staking Community Grants

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
I emailed you about content, but this page will also need a link to it. Probably from `/eth2/get-involved/`

- [ ] Add paragraph and link to grants page from `/eth2/get-involved/`
- [ ] Update with form link when available

File: src/content/eth2/get-involved/staking-community-grants/index.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+- Staking Education & Community
+- Staking & Validator Tools
+- Data Analysis & Visualizations
+
+But other topic areas are welcome too! In short, if you're working on cultivating the eth2 staking ecosystem, read on!
+
+<InfoBanner emoji=":thinking_face:">
+
+Wait, how do I get started with staking? Learn [what staking means](https://ethereum.org/en/eth2/), or try it first on the [Medalla testnet](https://medalla.launchpad.ethereum.org/)</a>
```
Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:31]:
```suggestion
Wait, how do I get started with staking? Learn [what staking means](https://ethereum.org/en/eth2/), or try it first on the [Medalla testnet](https://medalla.launchpad.ethereum.org/)
```

Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:31]:
```suggestion
Wait, how do I get started with staking? Learn [what staking means](/eth2/), or try it first on the [Medalla testnet](https://medalla.launchpad.ethereum.org/)</a>
```

File: src/content/eth2/get-involved/staking-community-grants/index.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+- The Ethereum Foundation runs a general grants initiative called the [Ecosystem Support Program](https://esp.ethereum.foundation/) (ESP).
+- At any time, you may submit an inquiry for support through ESP.
+- Thus, if you miss the deadline for this dedicated round of grants but have a proposal related to eth2 staking, head on over to [ESP](https://esp.ethereum.foundation).
+
+### How can I learn more about eth2?
+
+Here are some helpful resources:
+
+- [Ethereum 2.0 Overview](https://ethereum.org/en/eth2/) — [ethereum.org](http://ethereum.org)
```
Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:137]:
```suggestion
- [Ethereum 2.0 Overview](/eth2/) — [ethereum.org](/)
```

File: src/content/eth2/get-involved/staking-community-grants/index.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+- [The beacon chain](https://ethereum.org/en/eth2/the-beacon-chain/) — [ethereum.org](http://ethereum.org)
```
Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:138]:
```suggestion
- [The beacon chain](/eth2/the-beacon-chain/) — [ethereum.org](http://ethereum.org)
```

File: src/content/eth2/get-involved/staking-community-grants/index.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+- [eth2 quick update no. 19](https://blog.ethereum.org/2020/11/04/eth2-quick-update-no-19/) — _Danny Ryan_
+
+### How do I get started with staking and becoming a validator?
+
+- If you're new to staking, we encourage you to practice first on a testnet before moving to mainnet.
+
+  - Work through the [Medalla launchpad](https://medalla.launchpad.ethereum.org/) steps to set up a validator on the Medalla testnet.
+
+- Once you're ready for real ETH, head on over to the [Eth2 Launchpad](https://launchpad.ethereum.org/) to set up your eth2 validator.
+  - Remember to [check the deposit contract address](https://ethereum.org/en/eth2/deposit-contract/) before staking any ETH!
```
Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:153]:
```suggestion
  - Remember to [check the deposit contract address](/eth2/deposit-contract/) before staking any ETH!
```

File: src/content/eth2/get-involved/staking-community-grants/index.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+  - rumor is an interactive shell to run the Eth2 network stack, attach to testnets, debug clients, and extract data for tooling.
+- [wealdtech/chaind](https://github.com/wealdtech/chaind) for pulling historical state at a given node
+- eth2 block explorers
+  - [`Beaconcha.in`](https://beaconcha.in/)
+  - [`BeaconScan`](https://beaconscan.com/)
+  - [`BlockAction`](https://blockaction.io/)
+
+### How can I get more involved in eth2?
+
+- Visit [Get involved in Eth2](https://ethereum.org/en/eth2/get-involved/)
```
Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:171]:
```suggestion
- Visit [Get involved in Eth2](/eth2/get-involved/)
```

File: src/content/eth2/get-involved/staking-community-grants/index.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+# Eth2 Staking Community Grants
+
+_The Ethereum Foundation is sponsoring a wave of eth2 staking community grants. Proposals are due December 22, 2020. Here are all the details you need._
+
+<Divider />
+
+Calling all those interested in building up the Ethereum 2.0 staking and validator community!
+
+Ethereum 2.0 is happening! The mainnet deposit contract address is [deployed](https://ethereum.org/en/eth2/deposit-contract/). And [genesis](https://hackmd.io/@benjaminion/genesis) of the Beacon Chain is [right around the corner](https://blog.ethereum.org/2020/11/04/eth2-quick-update-no-19/). Which means, stakers are gearing up to stake.
```
Comment (samajammin) [inline src/content/eth2/get-involved/staking-community-grants/index.md:17]:
```suggestion
Ethereum 2.0 is happening! The mainnet deposit contract address is [deployed](/eth2/deposit-contract/). And [genesis](https://hackmd.io/@benjaminion/genesis) of the Beacon Chain is [right around the corner](https://blog.ethereum.org/2020/11/04/eth2-quick-update-no-19/). Which means, stakers are gearing up to stake.
```

**Comment by samajammin** [comment]
@gilbertginsberg FYI for next time - notice my commit here: https://github.com/ethereum/ethereum-org-website/pull/1754/commits/adf54f85a67d1989b9362a0b261c90662fb9b688

When linking to ethereum.org internally, please use relative links vs. full URL. This prevents links from refreshing the page.

There was also a stray closing a tag, which I believe was the cause of the broken build.

**Comment by gilbertginsberg** [comment]
> @gilbertginsberg FYI for next time - notice my commit here: [adf54f8](https://github.com/ethereum/ethereum-org-website/commit/adf54f85a67d1989b9362a0b261c90662fb9b688)
> 
> When linking to ethereum.org internally, please use relative links vs. full URL. This prevents links from refreshing the page.
> 
> There was also a stray closing a tag, which I believe was the cause of the broken build.

Ahh, okay okay. Will do @samajammin

**Comment by ryancreatescopy** [comment]
Closed. Page merged in #1766

**Pull request closed without merging**